src/perturbopy/postproc/calc_modes/spectral_cumulant.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
+import h5py as h5
+import matplotlib.pyplot as plt
+from perturbopy.postproc.calc_modes.calc_mode import CalcMode
+
+
+class SpectralCumulant(CalcMode):
+    """
+    A class to handle and analyze spectral functions computed using the cumulant expansion approach.
+
+    Attributes:
+    - T (numpy.ndarray): Array of temperatures corresponding to the spectral data.
+    - w (numpy.ndarray): Array of energy values (ω) in electron volts (eV).
+    - dw (float): Energy step size for the ω grid in eV.
+    - Akw (numpy.ndarray): Spectral function data, indexed by k-point, band, temperature, and ω.
+    """
+    def __init__(self, prefix):
+        f = h5.File(f"{prefix}_spectral_cumulant.h5", "r")
```

Comment:
We have a function that opens the HDF5 files in io_utils/io.py::open_hdf5
Please use this function. I know this might sound annoying but one day we decide to change something, it will be a lot easier if the usage of io functions is uniform throughout the codebase.